### ISSUE TITLE
HTML file suffix is already added

### ISSUE THREAD

**Issue by badcat** [GBratsos/brackets-zurb-foundation#11]
Not sure where this is coming from, but after creating a new Zurb Document then choosing File > Save, the file name is already suffixed with .html  -- So, the user would wind up with a file ready to save as "untitled.html.html"

How do we remove the first .html from the file name?

**Comment by GBratsos** [comment]
The plugin is predefined to save as html the file when you create a Zurb Foundation document. And in most cases that's the default functionality of a html template. So it's automated right now.

If you make the mistake with double .html.html you can rename your file afterwards.

**Comment by badcat** [comment]
Yes.. just seems like there should be a way to resolve the double suffix. I saw the original Bootstrap one did this too. Was hoping to have the save dialog only highlight the name of the file to be typed in and leave the (single) suffix alone. Like below from Visual Studio Code. 
![highlight](https://user-images.githubusercontent.com/26096/29128378-a18b2e8e-7cf1-11e7-8d6c-a9eadc7bec5b.png)
 I'll look into this.
